datumaro/plugins/data_formats/datumaro/exporter.py

```diff
@@ -303,6 +306,11 @@ def _convert_cuboid_3d_object(self, obj):
     def _convert_ellipse_object(self, obj: Ellipse):
         return self._convert_shape_object(obj)
 
+    def _convert_hashkey_object(self, obj):
+        converted = self._convert_annotation(obj)
+        converted.update({"hashkey": cast(obj.hash_key, np.array)})
+        return converted
```

Comment:
You don't need this because `self._save_hashkey_file()` will save hash key to the separate file.